### ISSUE TITLE
Gatsby Sharp

### ISSUE THREAD

**Issue by tlgimenes** [vtex/faststore#711]
Use gatsby sharp in one of our starters.

**Comment by lbebber** [comment]
doesn't this plugin do something that thumbor could/should handle?

**Comment by tlgimenes** [comment]
Yes, but we could actually just ditch thumbor and use sharp for small stores
